packages/client-core/src/world/reducers/scenes/service.ts

```diff
@@ -1,13 +1,13 @@
-import { Dispatch } from 'redux'
-import { client } from '../../../feathers'
+// import { Dispatch } from 'redux'
+// import { client } from '../../../feathers'
 
-export const createPublishProject =
-  (data) =>
```

Comment:
Please delete this file instead of commenting out if it's not going to be used now